index.js

```diff
@@ -140,15 +140,23 @@ function compileRunner(options) {
         }
       });
     }
+    var fixupPath = function(path) {
```

Comment:
Can you pull this out to a function outside of the `compileRunner` and give it JSDocs?